docs/tutorials/concepts/logging.md

```diff
@@ -3,7 +3,7 @@
 ## Enable the Default Logger
 
 OkapiLib has a [Logger](@ref okapi::Logger) class which is used internally by many of
-OkapiLib's classes. It is a good idea to use the [warn](@ref okapi::Logger::LogLevel::warn)
+OkapiLib's classes. It is a good idea to use the warn
```

Comment:
What good does this do?

Reply:
Here is how it looked before: 
![image](https://user-images.githubusercontent.com/16546293/71550171-eb636800-297e-11ea-9b27-ff16a12bd0f4.png)

With the line wrap the way it is, it makes it seem like the two words are part of the same link, which can be confusing.

When the user clicks on **warn**, it jumps to the anchor so that the rest of the enum is hidden:
![image](https://user-images.githubusercontent.com/16546293/71550213-ae4ba580-297f-11ea-9b82-9b88ae831423.png)

In my opinion, it is more user friendly when the user clicks on **LogLevel**, because then it puts the entire enum into view:

![image](https://user-images.githubusercontent.com/16546293/71550194-704e8180-297f-11ea-8c49-85dbe64f966d.png)


Only having one link will make it so the user will be navigated to where all the modes are shown, instead of just the warn enum, which has no additional information.